backend/src/store/room.go

```diff
@@ -251,6 +272,17 @@ func (r *Room) SetAutoReveal(autoReveal bool) {
 	r.EmitState()
 }
 
+func (r *Room) SetTimebox(timeboxInSeconds uint) {
+	r.mtx.Lock()
+	defer r.mtx.Unlock()
+	r.timeboxInSeconds = timeboxInSeconds
```

Comment:
Do I understand it right that when a timer is running the users cannot change the timebox so we don't restart the timer?

Reply:
yes. during a voting round the timer can not be changed. this is also prohibited in the UI. after a session is over a previous timer may still be running. if this is the case it is stopped when a new one is created by starting a new voting rount. whilst writing this i realized, that the timer should be stopped instead when the cards are revealed.

should the length of a timer be limited?

Reply:
I don't think so, but good question.